src/tool/cppwinrt/strings/base_collections_input_iterable.h

```diff
@@ -42,6 +42,12 @@ namespace winrt::impl
             return range_container<InputIt>{ m_begin, m_end };
         }
 
+#ifdef _DEBUG
+        void use_make_function_to_create_this_object() final
```

Comment:
Can we issue a static_assert and more friendly message instead?

Reply:
Sadly I haven't found a way to express this with a static_assert.